src/LiveComponent/assets/src/live_controller.ts

```diff
@@ -296,11 +296,26 @@ export default class extends Controller {
         }
     }
 
-    _makeRequest(action: string|null) {
+    _makeRequest(action: string|null, values: Array<string>|object) {
         const splitUrl = this.urlValue.split('?');
         let [url] = splitUrl
         const [, queryString] = splitUrl;
         const params = new URLSearchParams(queryString || '');
+        let valueParams = null;
+
+        if (Array.isArray(values)) {
+            valueParams = new URLSearchParams();
+
+            values.forEach((v, i) => valueParams.set(i, v));
+        } else if (typeof values === 'object' && Object.keys(values).length > 0) {
+            valueParams = new URLSearchParams();
+
+            Object.keys(values).forEach(k => valueParams.set(k, values[k]));
```

Comment:
If `values` if following this structure `{key: value}`, then you can use:
```suggestion
            valueParams = new URLSearchParams(values);
```

![image](https://user-images.githubusercontent.com/2103975/149283406-41eaf952-8d6d-4639-b1ac-2a785d2e765c.png)

Reply:
cool, forgot about that :)